### PR TITLE
Adapt code to openlayers version 3.18.1

### PR DESCRIPTION
**Pull request by fredj**

### CODE REVIEW

**Comment by sbrunner** [comment]
Looks good to be merged tomorrow :-)

**Comment by fredj** [comment]
@sbrunner can be merged now?

File: src/services/print.js [hunk 1]
```diff
@@ -710,7 +710,7 @@ ngeo.Print.prototype.encodeVectorStyleStroke_ = function(symbolizer, strokeStyle
   var strokeColor = strokeStyle.getColor();
   if (strokeColor !== null) {
     var strokeColorRgba = ol.color.asArray(strokeColor);
-    symbolizer.strokeColor = goog.color.rgbArrayToHex(strokeColorRgba);
+    symbolizer.strokeColor = goog.color.rgbArrayToHex(/** @type {Array.<number>} */ (strokeColorRgba));
```
Comment (gberaudo) [inline src/services/print.js]: Instead of a cast, what about using an instanceof `goog.asserts.assertInstanceof(strokeColorRgba, Array)`?

**Comment by gberaudo** [comment]
@fredj, I added a very minor remark. Feel free to merge with or without it.

**Comment by sbrunner** [comment]
OK for me to be merged :-)

**Pull request merged**